architecture/core_concepts/pods_and_services.adoc

```diff
@@ -650,12 +650,13 @@ labels:
 
 ====
 
+
+
 Consider:
 
-- A pod consisting of an *nginx* container, with the label
-*role=webserver*.
-- A pod consisting of an *Apache httpd* container, with the same label
-*role=webserver*.
+* A pod consisting of an *nginx* container, with the label *role=webserver*.
+
```

Comment:
Does this render correctly?

Reply:
@kalexand-rh Yes. What is your concern.

Reply:
There's an extra line between the list items - I forgot that asciidoc doesn't care. :)